### PR TITLE
[WIP] updated as FPWD

### PR DESCRIPTION
**Pull request by himorin**
for #12

- added date as XX which to be fixed
- added mention on modularization in status, but better to be in introduction?

### CODE REVIEW

File: index.bs [hunk 1]
```diff
@@ -17,6 +18,8 @@ Editor: Brandon Jones 87824, Google http://google.com/, bajones@google.com
 Editor: Nell Waliczek 93109, Amazon [Microsoft until 2018] https://amazon.com/, nhw@amazon.com
 
 Abstract: This specification module describes support for accessing button, trigger, thumbstick, and touchpad data associated with virtual reality (VR) and augmented reality (AR) devices on the Web.
+
+Status Text: This WebXR Gamepads Module is designed as a module to be loaded by <a href="https://www.w3.org/TR/webxr/">WebXR Device API</a>, and is originally included in WebXR Device API which was divided into core and modules.
```
Comment (NellWaliczek) [inline index.bs]: Can we change "module to be loaded by" to be "module to be implemented in addition to"

**Comment by NellWaliczek** [comment]
@himorin , is there anything further you need from us to get this PR unblocked?

**Comment by himorin** [comment]
PR just updated, sorry for late.

**Comment by himorin** [comment]
> @himorin , is there anything further you need from us to get this PR unblocked?

Once target date is fixed, I'll update this, and could be merged, I suppose.

File: index.bs [hunk 2]
```diff
@@ -17,6 +18,8 @@ Editor: Brandon Jones 87824, Google http://google.com/, bajones@google.com
 Editor: Nell Waliczek 93109, Amazon [Microsoft until 2018] https://amazon.com/, nhw@amazon.com
 
 Abstract: This specification module describes support for accessing button, trigger, thumbstick, and touchpad data associated with virtual reality (VR) and augmented reality (AR) devices on the Web.
+
+Status Text: This WebXR Gamepads Module is designed as a module to be implemented in addition to <a href="https://www.w3.org/TR/webxr/">WebXR Device API</a>, and is originally included in WebXR Device API which was divided into core and modules.
```
Comment (NellWaliczek) [inline index.bs]:
```suggestion
Status Text: This WebXR Gamepads Module is designed as a module to be implemented in addition to <a href="https://www.w3.org/TR/webxr/">WebXR Device API</a>, and was originally included in WebXR Device API which was divided into core and modules. 
```

Comment (NellWaliczek) [reply index.bs]: Let's fix this in a separate PR just to get publishing unblocked.

**Pull request merged**